buildkite/bazelci.py

```diff
@@ -426,8 +429,9 @@ class BuildkiteClient(object):
 uwoRMCwe
 """.strip()
 
-    _BUILD_STATUS_URL_TEMPLATE = ("https://api.buildkite.com/v2/"
-                                  "organizations/{}/pipelines/{}/builds/{}")
+    _BUILD_STATUS_URL_TEMPLATE = (
+        "https://api.buildkite.com/v2/" "organizations/{}/pipelines/{}/builds/{}"
```

Comment:
Nit: maybe removing this?

Reply:
I mean since they are in one line, we can just use one pair of double quote.

Reply:
The linter is stupid - fixed!